components/31-molecules/form-item/_form-item.scss

```diff
@@ -40,7 +40,7 @@
             align-items: flex-start;
 
             .field-message {
-              margin: 0 0 gutter()/2 .2rem;
+              margin: 0 0 gutter()*.5 .2rem;
```

Comment:
Add space between * and .5. Makes it clearer that it is a calculation instead of a wildcard.